code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -15,8 +15,8 @@
 	force_wielded = 20
 	throwforce = 5
 	throw_speed = 4
-	light_range = 5
-	light_power = 1
+	light_range = 7
+	light_power = 2
```

Comment:
buffing light_power beyond `1` makes the resulting light look pretty weird. this specific buff can be toned back